zomes/blog/entry_types/post/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "post",
+  "sharing": "public",
+  "links_to": [
+    {
+      "target_type": "post",
+      "tag": "comments"
+    }
+  ],
+  "anchored": [
+    {
+      "base": "App.Key.Hash",
+      "tag": "authored_posts"
+    }
```

Comment:
This implicitly defines link types that are being put onto ```App.Key.Hash``` pointing to posts under the tag "authored_posts". Can be retrieved like with ```get_links(agent, "authored_posts")```

Reply:
@lucksus and I talked about actually leaving anchors for the true anchor functionality that we want to merge into core, and convert this to `linked_from`